target/communication/sca_trigger_commands.py

```diff
@@ -0,0 +1,40 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""Communication interface for the SHA3 SCA application on OpenTitan.
+
+Communication with OpenTitan either happens over simpleserial or the uJson
+command interface.
+"""
+import json
+import time
+from typing import Optional
+
+
+class OTTRIGGER:
+    def __init__(self, target, protocol: str) -> None:
+        self.target = target
+        self.simple_serial = True
+        if protocol == "ujson":
+            self.simple_serial = False
+
+    def select_trigger(self, trigger_source: Optional[int] = 0):
+        """ Select the trigger source for SCA.
+        Args:
+            trigger_source:
+                            - 0: Precise, hardware-generated trigger - FPGA only.
+                            - 1: Fully software-controlled trigger.
+        """
+        if self.simple_serial:
+            self.target.write(cmd="t", data=bytearray([trigger_source]))
+        else:
+            # TODO: without the delay, the device uJSON command handler program
+            # does not recognize the commands. Tracked in issue #256.
```

Comment:
nit: We usually use this format for TODO. Makes it easier to search the repository.

`TODO(#256): Without the delay, the device uJSON command handler program does not recognize the commands.`

Reply:
Good point raised by @vrozic. Please make sure to align the format before merging.

Reply:
Hm I followed the recommendation of the OpenTitan [C style guide](https://opentitan.org/book/doc/contributing/style_guides/c_cpp_coding_style.html#todo-comments) regarding comments. There seems not to be a recommendation for Python though.

But I am following @vrozic advise and change it to `TODO(#256)` for now and I created an issue that we can streamline this later on.